aleph/index/entities.py

```diff
@@ -66,19 +64,31 @@ def iter_entities(
     includes=PROXY_INCLUDES,
     excludes=None,
     filters=None,
-    should=None,
     sort=None,
-    raise_on_error=True,
+    randomize=False,
+    random_seed=None,
```

Comment:
@mynameisfiber May be we should get rid of these arguments too since they are not used anywhere anymore?